src/components/dictionaryConcepts/components/CreateMapping.jsx

```diff
@@ -89,10 +91,6 @@ class CreateMapping extends Component {
               </div>
             </div>
           )}
-        </td>
-
-        <td className="react-async">
```

Comment:
Can you describe how this change is related to the ticket?

Reply:
@dkayiwa  I am moving the code field from the previous column (`<td>`) to the next column which is on line 94 of this file on the `master` branch: https://github.com/openmrs/openmrs-ocl-client/blob/master/src/components/dictionaryConcepts/components/CreateMapping.jsx 

That's just how the file differences were registered but I did not actually delete that line.